docs/source/administrator/upgrading.md

```diff
@@ -1,5 +1,3 @@
-(upgrading)=
```

Comment:
This label has the same name as the page. When I tried to use it I got this warning:
```
WARNING: more than one target found for 'myst' cross-reference upgrading: could be :std:ref:`upgrade guide` or :std:doc:`upgrade guide`
```

Reply:
I don't know, but I wonder how many uses we have of the link? The labels can be renamed etc freely without risk of breaking something I think.